metagraph-plugin/metagraph_katana/plugins/metagraph_katana/translators.py

```diff
@@ -48,12 +48,24 @@ def networkx_to_katanagraph(x: NetworkXGraph, **props) -> KatanaGraph:
 
 @translator
 def katanagraph_to_networkx(x: KatanaGraph, **props) -> NetworkXGraph:
+    pg = x.value
+    dest_list = [dest for src in pg for dest in [pg.get_edge_dest(e) for e in pg.edges(src)] ]
+    for nid in pg:
+        if pg.edges(nid) == range(0,0):
+            assert nid in dest_list, "NetworkX does not support graph with isolated nodes, please use a cleaned Katana Graph"
```

Comment:
Since "cleaned graph" is not well defined, I don't think the second part is useful. Just say "NetworkX does not support graph with isolated nodes". Also you should raise `ValueError` instead of using assert. Assert means "something unexpected as failed" in this case it's actually kind of expected. The user passed a value that cannot be used in this way.

Please apply the same changes to the assert in the next loop.